static/js/rest.js

```diff
@@ -29,11 +31,11 @@ export type RestState = {
   fetchStatus?: string,
 };
 
-const INITIAL_STATE: RestState = {
+const INITIAL_REST_STATE: RestState = {
```

Comment:
definitely useful, but we don't even have types declared for the 2 REST actions we've defined, so i'm inclined to implement this in some other PR